README.md

```diff
@@ -17,3 +17,34 @@ Our goal here is to write a web-based ClojureScript tool that will allow
 students to check the correctness of each step in their proofs, at least for
 simple propositional calculus. It's likely to be _very_ simple, at least until
 there's some evidence that the students find it useful.
+
+## Quick Start
+
+:warning: edsger is a still under active development. The running instructions are most likely to change in the future.
+
+Clone and run this repository to see the app in action
+```
+git clone https://github.com/NicMcPhee/edsger
+cd edsger
+lein figwheel
+```
+
+## Usage
+
+When you validate a logic expression looking:  
+```
+  a ∧ (c ∨ b)
+≣    <(3.24) Symmetry of ∨>
+  a ∧ (b ∨ c) 
+```
+you need to type the expressions and rules in pseudo-clojure style.
+```
+first box  : (and a (or c b))
+second box : (or a b)
+third box  : (or b a)
+fourth box : (and a (or b c))
+```
+And, click the **validate** button to check your reasoning :100:.
+
```

Comment:
You might as well mention that changing the input format is the very next thing we are working on.